jstests/ssl/x509/mkcert.py

```diff
@@ -26,8 +27,10 @@
 
 CONFIG = Dict[str, Any]
 
-MUST_STAPLE_KEY = b'1.3.6.1.5.5.7.1.24'
-MUST_STAPLE_VALUE = str('DER:30:03:02:01:05').encode('utf-8')
+MUST_STAPLE_KEY_STR = '1.3.6.1.5.5.7.1.24'
+MUST_STAPLE_KEY = bytes(MUST_STAPLE_KEY_STR, "utf-8")
+MUST_STAPLE_VALUE_STR ='DER:30:03:02:01:05'
```

Comment:
I know this commit isn't adding it, but could either of you document what SEQ: [ 1, NULL ] actually signifies? (Also that I'm decoding that correctly?)